Form-Controls/styles.css

```diff
@@ -0,0 +1,288 @@
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    margin: 0 10px;
+    box-shadow: rgba(171, 160, 160, 0.24) 0px 3px 8px;
+    border-radius:  10px;
+    border: none;
+    gap: 5px;
+}
+
+.dress-size {
+    width: 80px;
+    height: 80px;
+}
+
+.dress-sizeg {
+    width: 130;
+    height: 80;
+}
+
+.bl-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 1rem;
+    border-radius: 10px;
+    background-color: var(--pink);
+    margin: .5rem;
+    padding: 1rem;
+}
+.b-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 1rem;
+    border-radius: 10px;
+    background-color: var(--blue);
+    margin: .5rem;
+    padding: 1rem;
+}
+.r-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 1rem;
+    border-radius: 10px;
+    background-color: var(--blue);
+    margin: .5rem;
+    padding: 1rem;
+}
+input[type="text"] {
+    width: 300px;
+    border-radius: 10px;
+    padding: 0 1rem;
+    height: 30px;
+    border: 1px solid var(--blue-green);
+}
+input[type="email"] {
+    width: 300px;
+    border-radius: 10px;
+    padding: 0 1rem;
+    height: 30px;
+    border: 1px solid var(--gray);
+}
+
+input[type="date"] {
+    width: 300px;
+    border-radius: 10px;
+    padding: 0 2rem;
+    height: 30px;
+    border: 1px solid var(--pink);
+}
+
+input[type="radio"] {
+    accent-color: var(--purple);
+}
+input[type="radio"]:hover {
+    accent-color: var(--pink);
+}
+.sizecontainer {
+    grid-area: 2 / 1 / 3 / 3;
+    display: grid;
+    grid-template-columns: repeat(2, 40px);
+    grid-template-rows: repeat(7,20px);
+    gap: 5px;
+    justify-content: center;
+    margin: 0 10px;
+    padding-top: 20px;
+    box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
+    border-radius: 10px;
+    border: none;
+    background-color: var(--aqua);
+    padding:  1rem;
+}
+
+.date-container {
+    grid-area: 3 / 1 / 4 / 2;
+    box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
+    border-radius: 10px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 10px;
+    margin: 0 10px;
+    border: none;
+    background-color: var(--gray);
+    padding: 0 2rem;
+}
+
+
+legend {
+    margin-left: 30px;
+    color:purple;
+    
+}
+button {
+    padding: 1rem 1rem;
+    border: none;
+    background-color: var(--green);
+    box-shadow: rgba(232, 72, 72, 0.24) 0px 3px 8px;
+    border-radius: 20px;
+    color:#f17ed2;
+    transition: all .4s ease-in-out;
+    font-family: 'Poor Story';
+    font-size: 24px;
+}
+button:hover {
+    background-color: var(--blue-green);
+}
+
+button:focus {
+    box-shadow: none;
+}
+
+/* footer */
+
+footer {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--purple);
+    padding: 1rem;
+    color: navy;
+}
+
+
+@media screen and (max-width: 880px) {
```

Comment:
This is overriding the default styles on small screens up to 880px wide. Did you mean to apply these styles when the viewport is bigger than 880px wide?
The best approach for media queries is 'mobile first' -
1. write default CSS with no media query, for small screens
2. add one or more media queries using 'min-width' for styles to be applied when the viewport is bigger